### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by ReadmeCritic**
Created with https://github.com/dkhamsing/frankenstein

### CODE REVIEW

**Comment by ReadmeCritic** [comment]
I signed it

File: README.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ If you experience an issue with Service Worker support in your application, chec
 
 * [sw-toolbox](https://github.com/GoogleChrome/sw-toolbox/issues)
 * [platinum-sw](https://github.com/PolymerElements/platinum-sw/issues)
-* [platinum-push-notifications-manager](https://github.com/PolymerElements/push-notification-manager/)
+* [platinum-push-notifications-manager](https://github.com/PolymerElements/platinum-push-messaging)
```
Comment (samccone) [inline README.md]: zomg it found redirects!

this is a coooooool tool

**Comment by samccone** [comment]
wow this is awesome :+1:

**Comment by dkhamsing** [comment]
:smile:

**Pull request merged**